extensions/mongodb-client/deployment/src/main/java/io/quarkus/mongodb/deployment/DevServicesBuildTimeConfig.java

```diff
@@ -45,4 +45,30 @@ public class DevServicesBuildTimeConfig {
     @ConfigItem
     public Map<String, String> containerEnv;
 
+    /**
+     * Indicates if the Mongo server managed by Quarkus Dev Services is shared.
```

Comment:
Could you make sure you use `MongoDB` instead of `Mongo` in the javadoc of the configuration properties? They are published on the website.
(Note that other properties might not comply but let's not add more :))

Reply:
Sure, will fix it. Naming is so hard ;)